R/plumbertableau-package.R

```diff
@@ -0,0 +1,2 @@
+#' @keywords internal
+"_PACKAGE"
```

Comment:
I'm not sure if I understand why we need this

Reply:
With this, you can type `?plumbertableau` at the R command and see package help. It's important for onboarding new users who might not know anything other than the package name.

Reply:
Specifically, this is what `?plumbertableau` looks like:
<img width="607" alt="Screen Shot 2021-07-20 at 1 09 42 PM" src="https://user-images.githubusercontent.com/3117884/126366523-9a6bb0a2-a9f1-49c8-8e5c-a61e5a881e25.png">

Reply:
TIL. Sounds great!